tutorials/plot_background_statistics.py

```diff
@@ -0,0 +1,566 @@
+plot_t_p(ts[-1], ps[-1], titles[-1], mccs[-1])
+
+###############################################################################
+# Visualize and compare methods
+# -----------------------------
+# Let's take a look at these statistics. The top row shows each test statistic,
+# and the bottom shows p-values for various statistical tests, with the ones
+# with proper control over FWER or FDR with bold titles.
+
+fig = plt.figure(facecolor='w', figsize=(14, 3))
+assert len(ts) == len(titles) == len(ps)
+for ii in range(len(ts)):
+    ax = [fig.add_subplot(2, 10, ii + 1, projection='3d'),
+          fig.add_subplot(2, 10, 11 + ii)]
+    plot_t_p(ts[ii], ps[ii], titles[ii], mccs[ii], ax)
+fig.tight_layout(pad=0, w_pad=0.05, h_pad=0.1)
+plt.show()
+
+###############################################################################
+# The first three columns show the parametric and non-parametric statistics
+# that are not corrected for multiple comparisons:
+#
+# - **t-test** has jagged edges.
+# - **"hat" variance correction** of the t-test has reduced peaky edges,
+#   correcting for sharpness in the statistic driven by low-variance voxels.
+# - **non-parametric resampling test** is very similar to the t-test, as
+#   the data are parametric.
+#
+# The next three columns show multiple comparison corrections of the
+# mass univariate tests (parametric and non-parametric). These
+# too conservatively correct for multiple comparisons because neighboring
+# voxels in our data are correlated:
+#
+# - **Bonferroni correction** eliminates any significant activity.
+# - **FDR correction** is less conservative than Bonferroni.
+# - **Permutation test with a maximum statistic** also eliminates any
+#   significant activity.
+#
+# The final four columns show the non-parametric, cluster-based permutation
+# tests with a maximum statistic:
+#
+# - **Standard clustering** identifies the correct region. However, the whole
+#   area must be declared significant, so no peak analysis can be done.
+#   Also, the peak is broad.
+# - **Clustering with "hat"** tightens the estimate of significant activity.
+# - **Clustering with TFCE** allows analyzing each significant point
+#   independently, but still has a broadened estimate.
+# - **Clustering with TFCE and "hat"** tightens the area declared
+#   significant (again FWER corrected).
+#
+# Statistical functions in MNE
+# ----------------------------
+# MNE provides several convenience parametric testing functions that can be
+# used in conjunction with the non-parametric clustering methods. However,
+# the set of functions we provide is not meant to be exhaustive.
+#
+# If the univariate statistical contrast of interest to you is not listed
+# here (e.g., interaction term in an unbalanced ANOVA), consider checking
+# out the :mod:`statsmodels` package. It offers many functions for computing
+# statistical contrasts, e.g., :func:`statsmodels.stats.anova.anova_lm`.
+# To use these functions in clustering:
```

Comment:
Maybe there should be a full example here, perhaps for one of the statsmodels tests.

Reply:
I'm hoping @cbrnr will add one later, and we can link to it. I think including it here might be a bit much for this intro doc. We might want a separate one: "which statistical test should I use for my data?" (even in a simple univariate case)